File: Master3a/Master3a/ViewController.m
//
//  ViewController.m
//  Master3a
//
//  Created by Davide Morelli on 21/11/14.
//  Copyright (c) 2014 com.biobeats. All rights reserved.
//

#import "ViewController.h"
#import "AppDelegate.h"
#import "Categorie.h"

@interface ViewController ()
@property (strong, nonatomic) IBOutlet UILabel *myLabel;

@end

@implementation ViewController
- (IBAction)transpose:(id)sender {
    [UIView animateWithDuration:5.0 animations:^{
        _myLabel.alpha = 0.0;
    }];
}
- (IBAction)rotate:(id)sender {
    [UIView animateWithDuration:5.0 animations:^{
        _myLabel.alpha = 1.0;
    }];
}

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    

}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: testPicture/testPicture/ViewController.swift
//
//  ViewController.swift
//  testPicture
//
//  Created by Davide Morelli on 06/02/15.
//  Copyright (c) 2015 BioBeats. All rights reserved.
//

import UIKit

class ViewController: UIViewController, UIImagePickerControllerDelegate, UINavigationControllerDelegate {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


    @IBOutlet weak var img: UIImageView!
    @IBAction func takePicture(sender: AnyObject) {
        var controller = UIImagePickerController()
        controller.delegate = self
        controller.allowsEditing = true
        controller.sourceType = UIImagePickerControllerSourceType.Camera
        self.presentViewController(controller, animated: true) { () -> Void in
            //
        }
    
    }
    
    func imagePickerController(picker: UIImagePickerController!, didFinishPickingImage image: UIImage!, editingInfo: [NSObject : AnyObject]!) {
        NSLog("l'utente ha fatto la foto")
        self.img.image = image
        picker.dismissViewControllerAnimated(true, completion: { () -> Void in
            //
        })
    }
    
    func imagePickerControllerDidCancel(picker: UIImagePickerController) {
        NSLog("l'utente ha annullato")
        picker.dismissViewControllerAnimated(true, completion: { () -> Void in
            //
        })
     }


}

